### ISSUE TITLE
[Request] Alternative (local) address for an entry

### ISSUE THREAD

**Issue by filcuk** [bastienwirtz/homer#601]
**Is your feature request related to a problem? Please describe.**
I would love to have the option to include alternative (local) address for each service.

**Describe the solution you'd like**
I would like to have 'main' (usually domain) and an alternative address (usually local).
This could, for example, be accessed by corner hover on desktop, or swipe on mobile.

**Describe alternatives you've considered**
The alternative is to add a separate service entry.
Best option currently may be to maintain a mirror page which uses local urls where applicable.

---

I do understand if this isn't something you would like to build into Homer.
It aims to be simple and that's largely why it's great.

**Comment by henry5001** [comment]
Yes, I also want config.yml to support local URL configuration, I have a requirement to add items to [Android arm64 Termux](https://github.com/termux/termux-app), it seems that config.yml does not yet support dynamic LAN IP, because every time the phone changes a LAN environment, it may become a new LAN IP, if I want to add a new web service port in config.
